lua/layers/mode.lua

```diff
@@ -24,6 +24,7 @@
 ---@class layers.mode
 ---@field _win integer
 ---@field _maps table<layers.mapmode, layers.keymaps>
+---@field _mode_name string
```

Comment:
```suggestion
---@field _mode_name string?
```

Needed so that below usage does not cause a lua_ls diagnostic.